### ISSUE TITLE
Authenticate users using alternate method?

### ISSUE THREAD

**Issue by henrytsai** [yasyf/hifromtheotherside#26]
I've had maybe 4 people write to me saying they don't use facebook or deactivated it because election made them sad. What are other reliable ways of authenticating people? Not proposing that we do that now, but want to know wha the options are.

**Comment by dmitryzhd** [comment]
I think that's pretty much it bc no one uses google+ either. The other
option is to just have a regular sign up process where you fill out your
demographics in a form

**Comment by yasyf** [comment]
We could do login with Google account, that might cover some of the stragglers

**Comment by yasyf** [comment]
(In addition to fb)

**Comment by yasyf** [comment]
I'm hesitant to make people create another login for this and don't want to keep passwords

**Comment by henrytsai** [comment]
Agreed on not keeping passwords - if we offered a Google login, what are the downsides? If people somehow logged in with both, but have the same email associated with both services, would we deduplicate?

**Comment by henrytsai** [comment]
Let's do LinkedIn auth as a second option. I'm surprised at how many people don't use Facebook.
